lenovo/thinkpad/general.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, ... }:
+
+with lib;
+
+{
+  hardware.trackpoint = mkDefault {
+    enable = true;
+    emulateWheel = true;
+  };
+
+  hardware.enableAllFirmware = true;
+  services.tlp.enable = true;
+
+  services.xserver = {
+    synaptics.enable = false;
+    libinput.enable = true;
```

Comment:
libinput is questionable. I disabled it again on my machine, because it was buggy.

Reply:
well, I have to admit that I'm currently using synaptics as well (because I didn't manage to get kinetic scrolling working, which is perfectly fine with synaptics).
However synaptics is deprecated in NixOS, so I changed it here and will try a migration to libinput on my machine in the next days...

Reply:
@Profpatsch do you think we should work with `mkDefault false`?